source/agora/crypto/Key.d

```diff
@@ -20,6 +20,7 @@ import agora.crypto.ECC;
 import agora.crypto.Hash;
 import agora.crypto.Schnorr;
 import agora.serialization.Serializer;
+import agora.utils.WellKnownHRPs : getWellKnownHRP;
```

Comment:
No, we should absolutely not have a dependence between this code and `WellKnownHRPs`.
The well-known keys (and HRP) are test utilities only. They can depend on the production code, but the production code can NEVER depend on them.

Reply:
I see what you mean. I will find another way soon. Thanks.
However, I only made human-readable parts for [a .. zz] in `WellKnownHRPs` because if I make all the HRPs for [a .. azz], it makes the testing time increase 40%. I don't know why for sure but only know that the very long AA for HRPs causes it. If you have any ideas or suggestions for that, please tell me about that.